src/transform.c

```diff
@@ -20,7 +20,9 @@
 
 #include "internal.h"
 
+#ifdef __APPLE__
 #include <libkern/OSByteOrder.h>
```

Comment:
this header is [opensource](http://www.opensource.apple.com/source/xnu/xnu-792.13.8/libkern/libkern/OSByteOrder.h) and all of the implementations of OSSwap* are contained in the header, we should figure out if we can import it into the repo (or re-implement it, it's pretty trivial)